### PR TITLE
school library: add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by mrcbq**
Hello, In this opportunity I did the following:

- [x] Create a class Person with the following:

  Instance vars: @id, @name, and @age.
  Constructor with name, age, and parent_permission as parameter. name and parent_permission are optional and have default values of "Unknown" and true.
  Getters for @id, @name, and @age.
  Setters for @name and @age.
  Private method of_age? that returns true if @age is greater or equal to 18 and false otherwise.
  Public method can_use_services? that returns true if person is of age or if they have permission from parents.

- [x] Create class Student with the following:

  Inherits from Person.
  Constructor extends parent's constructor by adding @classroom and a parameter for it.
  Method play_hooky that returns "¯\(ツ)/¯".

- [x] Create class Teacher with the following:

  Inherits from Person.
  Constructor extends parent's constructor by adding @specialization and a parameter for it.
  Override can_use_services? so it always returns true.
  Each class should be saved in a separate file.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_reader :id, :name, :age
```
Comment (rloterh) [inline person.rb]: - [x] Let's encapsulate _name_ and _age_ variables _with **attr_accessor**_ as we intend to both and read and modify their values.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+**I would like to thank to Microverse for giving this inspiring project**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (rloterh) [inline README.md]: - [x] MIT License is broken, let's ensure to fix it to make our ReadMe stand out even better!

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Marco, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁 🚦 🌴  🏃 🌳 🏡 🏘️ 🌲 

## Highlights 👍  
- Good job so far implementing object associations!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
Kindly include appropriate instructions in your ReadMe on how to run your app, under the _Getting Started_ section.
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (mrcbq) [reply person.rb]: It's Done.

File: README.md (hunk 2 above)
Comment (mrcbq) [reply README.md]: It's Done!

**Review by franclobo — APPROVED** [review]
Hi @mrcbq, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️
- Descriptive PR ✔️
- Correct Git flow ✔️
- Correct classes ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**